sdk/python/kfp/components/_components.py

```diff
@@ -76,6 +76,12 @@ def load_component_from_url(url):
     '''
     if url is None:
         raise TypeError
+
+    #Handling Google Cloud Storage URIs
```

Comment:
Adding a TODO to handle private gs URI.

Reply:
I think I'll just implement the support (in the next PR).